specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2016-10-01/keyvault.json

```diff
@@ -564,6 +1011,56 @@
       ],
       "x-ms-azure-resource": true
     },
+    "VaultPatchParameters": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags that will be assigned to the key vault. "
+        },
+        "properties": {
+          "$ref": "#/definitions/VaultPatchProperties",
+          "description": "Properties of the vault"
+        }
+      },
+      "description": "Parameters for creating or updating a vault",
+      "x-ms-azure-resource": true
+    },
+    "VaultAccessPolicyParameters": {
+      "properties": {          
+          "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource id of access policy."
+        },
+          "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource name of access policy."
+        },
+          "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource name of access policy."
+        },
+          "location": {
```

Comment:
Why is location readOnly? Is this a tracked resource or proxy only?

Reply:
This is a proxy to access policies which are defined on a vault which is a tracked resource.  The value for location is derived from the vault and not directly settable.

Reply:
lets remove the location property then.